README.md

```diff
@@ -1,2 +1,230 @@
+* `strategy`: Specify which training strategy should be used (more details below).
+* `architecture`: Specify which architecture should be used. (more details below)
+* `tensorboard`: [Optional] If specified, will spawn an instance of TensorBoard to monitor your trainings
+  * `log_dir`: Directory where the summaries are saved.
+  * `pvc_name`: Name of an existing `PermanentVolumeClaim` that should be mounted.
+  * `public`: If set to `True` then a public IP will be created for TensorBoard (provided your Kubernetes cluster supports this). Otherwise only a private IP will be created. -->
+
+<!-- ### Training Strategies -->
+
+#### Simple Training
+
+Your class needs to define a `train` method that will be called during training:
+
+```python
+from fairing.train import Train
+
+@Train(repository='<your-repo-name>')
+class MyModel(object):
+    def train(self):
+      # Training logic goes here
+
+```
+<!-- No `strategy` is specified here, since the default `strategy` is `basicTrainingStrategy`. -->
+
+Complete example: [examples/simple-training/main.py](./examples/simple-training/main.py)
+
+
+#### Hyperparameters Tuning
+
+Allows you to run multiple trainings in parallel, each one with different values for your hyperparameters.
+
+Your class should define a `hyperparameters` method that returns an dictionary of hyperparameters and their values.
+This dictionary will be automatically passed to your `train` method. 
+Don't forget to add a new argument to your `train` method to received the hyperparameters.
+
+```python
+from fairing.train import Train
+from fairing.strategies.hp import HyperparameterTuning
+
+@Train(
+    repository='<your-repo-name>',
+    strategy=HyperparameterTuning(runs=6),
+)
+class MyModel(object):
+    def hyperparameters(self):
+      return {
+        'learning_rate': random.normalvariate(0.5, 0.45)
+      }
+
+    def train(self, hp):
+      # Training logic goes here
+```
+
+To specify that we wanted to train our model using hyperparameters tuning, and not just a simple training, 
+we passed a new `strategy` parameter to the `@Train` decorator, and specified the number of runs we wish to be created.
+
+
+Complete example: [examples/hyperparameter-tuning/main.py](./examples/hyperparameter-tuning/main.py)
+
+#### Population Based Training
+
+We can also ask `fairing` to train our code using [Population Based Training](https://deepmind.com/blog/population-based-training-neural-networks/).
+
+This is a more advanced training strategies that needs hook into different lifecycle steps of your model, thus we need to define several additional method into our model class.
+
+A multiple read/write PVC name needs to be pass to the `PopulationBasedTraining` strategie. This is used to store and exchange the different models generated by our training to enable the `explore/exploit` mechanism of Population Based Training.
```

Comment:
Can you explain why the PVC is needed?

Reply:
I was thinking that fairing would be responsible for converting the notebook to a container, and then spawning the correct TF/Pytorch job through the TFJob CR.